### ISSUE TITLE
SOCKS5 TCP Support? 

### ISSUE THREAD

**Issue by osousa** [blechschmidt/tun2proxy#92]
Hi there, 

i am trying to use it, but i'm getting the following:

`zero3:tun2socks:# ./tun2proxy --tun tun0 --verbosity trace --dns direct --proxy SOCKS5://user:pass@1.2.3.4:9090
[2024-02-22T01:09:03Z INFO  tun2proxy::desktop_api] Proxy socks5 server: 1.2.3.4:9090
[2024-02-22T01:09:03Z TRACE tun2proxy] Unknown transport
[2024-02-22T01:09:03Z TRACE tun2proxy] Session count 1
[2024-02-22T01:09:03Z INFO  tun2proxy] Beginning #0 UDP [fe80::386f:4444:6e9d:1111]:5355 -> [ff02::1:3]:5355
[2024-02-22T01:09:03Z TRACE tun2proxy] Unknown transport
[2024-02-22T01:09:04Z ERROR tun2proxy] #0 UDP [fe80::386f:4444:6e9d:1111]:5355 -> [ff02::1:3]:5355 error "SOCKS connection failed: Reply::GeneralFailure"
[2024-02-22T01:09:04Z TRACE tun2proxy] Session count 0
[2024-02-22T01:09:04Z TRACE ipstack] channel closed
`
I see UDP as a feature, thought TCP would be implicit, but probably it only supports UDP , right?

Appreciated,
Cheers!

**Comment by ssrlive** [comment]
No, both supported, depends on your SOCKS5 server. read the [wiki](https://github.com/blechschmidt/tun2proxy/wiki#linux)
